.github/workflows/nodejs.yml

```diff
@@ -1,24 +1,18 @@
 name: Node.js CI
-
 on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [14.x, 15.x]
-
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout
+      uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - run: npm run prod_build --if-present
-    - run: npm test
-      env:
-        CI: true
```

Comment:
See [here](https://docs.github.com/en/actions/reference/environment-variables) for details